docs/dev/table/connect.md

```diff
@@ -164,21 +164,8 @@ CREATE TABLE MyUserTable (
   'connector.properties.zookeeper.connect' = 'localhost:2181',
   'connector.properties.bootstrap.servers' = 'localhost:9092',
 
-  -- specify the update-mode for streaming tables
```

Comment:
Change above "The connector might already provide a fixed format with fields and schema." to "The connector might already provide a fixed format."?